storage/rocksdb/rdb_native_dd.h

```diff
@@ -20,15 +20,22 @@
 
 /* MySQL header files */
 #include "sql/dd/object_id.h"
+#include "sql/handler.h"
 
 namespace dd {
 class Table;
 }
 
 namespace myrocks {
 
+constexpr const char *rocksdb_dd_space_name = "mysql";
```

Comment:
```suggestion
constexpr const char rocksdb_dd_space_name[] = "mysql";
```

The pointer version works, but its sizeof is 8, which is very annoying to find